android/app/src/main/java/io/coursepick/coursepick/presentation/course/CoursesViewModel.kt

```diff
@@ -158,7 +171,7 @@ class CoursesViewModel
 
                     _state.value =
                         state.value?.copy(
-                            originalCourses = courseItems,
+                            originalCourses = courseItems.map { CourseListItem.Course(it) },
```

Comment:
```suggestion
                            originalCourses = courseItems.map(CourseListItem::Course),
```
이렇게 표현할 수도 있겠네요. 아니라면 인자 명시를 추가해주세요! 👍

Reply:
refactor: convention 준수

반영했습니당!